build.gradle.kts

```diff
@@ -6,28 +6,23 @@ plugins {
     idea
     kotlin("jvm") version "2.0.0"
     id("maven-publish")
-    id("com.teamresourceful.resourcefulgradle") version "0.0.+"
-    id("net.neoforged.moddev") version "0.1.126"
+    alias(libs.plugins.resourcefulgradle)
+    alias(libs.plugins.moddev)
 }
 
-val minecraftVersion: String by project
 val modId = "techarium"
 
 base {
-    archivesName.set("$modId-$minecraftVersion")
+    archivesName.set("$modId-${libs.versions.minecraft.get()}")
```

Comment:
Use a mapped provider instead of resolving the provider immediately. (libs.versions.minecraft.map)